1-JavaScript/Week3/homework/ex7-mindPrivacy.js

```diff
@@ -30,7 +30,9 @@ const employeeRecords = [
 ];
 
 // ! Function under test
-function filterPrivateData(/* TODO parameter(s) go here */) {
+function filterPrivateData(employeeRecords) {
+  const nonPrivateRecords = employeeRecords.map((employeeRecords) => ({name: employeeRecords.name, occupation: employeeRecords.occupation, email: employeeRecords.email,}));
```

Comment:
Nice work! We can also use object destructuring to save some typing

```suggestion
  const nonPrivateRecords = employeeRecords.map(({ name, occupation, email }) => ({name, occupation, email}));
```